ray-operator/apis/ray/v1alpha1/raycluster_types.go

```diff
@@ -111,6 +111,8 @@ type RayClusterStatus struct {
 	// LastUpdateTime indicates last update timestamp for this cluster status.
 	// +nullable
 	LastUpdateTime *metav1.Time `json:"lastUpdateTime,omitempty"`
+	// Service Node Ports
```

Comment:
Let's just say service endpoints because node port is just one of the implementation. It could be public LB endpoint as well